flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/tiered/netty/NettyPayloadQueue.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.io.network.partition.hybrid.tiered.netty;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+/**
+ * The {@link NettyPayloadQueue} is the queue to contain different types of {@link NettyPayload} and
+ * calculate the backlog number.
+ */
+public class NettyPayloadQueue {
```

Comment:
Basically, this should impl `Queue`.

Reply:
The interface `java.util.Queue` extends an interface `java.util.Collection`. The interface `java.util.Collection` also extends another interface `java.lang.Iterable`. There're many methods in these three interfaces that `NettyPayloadQueue` doesn't need in fact.  I'd prefer it not to implement any interface. WDYT? 🤔

Reply:
This is because our name is confusing, making it difficult for other developers to imagine an `XxxQueue` that does not implement the interface of `Queue`. Perhaps you can rename the class to `NettyPayloadManager` or anything else.

Reply:
`NettyPayloadManager` has been renamed to `NettyPayloadQueue`.